src/navigation/navigationTabs/__test__/navigationTabsMockUps.js

```diff
@@ -0,0 +1,68 @@
+export const items = [
```

Comment:
Used for unit testing and some tabs docs, I don't think we should reference files outside the docs folder in the DOCS code snippets but I already asked about that previously in [here](https://github.com/saddlebackdev/react-cm-ui/pull/195/files#r477452564), please let me know, thanks.